### PR TITLE
feat: delete segment from CR

### PR DESCRIPTION
**Pull request by sjaanus**

### CODE REVIEW

File: frontend/src/component/changeRequest/ChangeRequest/ChangeRequest.tsx [hunk 1]
```diff
@@ -55,7 +63,7 @@ export const ChangeRequest: VFC<IChangeRequestProps> = ({
                     {feature.changes.map((change, index) => (
                         <FeatureChange
                             key={index}
-                            discard={
+                            actions={
```
Comment (sjaanus) [inline frontend/src/component/changeRequest/ChangeRequest/ChangeRequest.tsx:66]: Renaming old confusing prop name

File: frontend/src/component/changeRequest/ChangeRequest/Changes/Change/ChangeActions.tsx [hunk 2]
```diff
@@ -60,7 +57,7 @@ const StyledPopover = styled(Popover)(({ theme }) => ({
 export const ChangeActions: FC<{
     changeRequest: IChangeRequest;
     feature: string;
-    change: IFeatureChange;
+    change: IChange;
```
Comment (sjaanus) [inline frontend/src/component/changeRequest/ChangeRequest/Changes/Change/ChangeActions.tsx:60]: Now can take in all type of changes

File: frontend/src/component/changeRequest/ChangeRequest/ChangeRequest.tsx [hunk 3]
```diff
@@ -29,11 +29,19 @@ export const ChangeRequest: VFC<IChangeRequestProps> = ({
...
                 <SegmentChange
-                    key={segment.payload.id}
-                    segmentChange={segment}
+                    key={segmentChange.payload.id}
+                    segmentChange={segmentChange}
                     onNavigate={onNavigate}
+                    actions={
+                        <ChangeActions
+                            changeRequest={changeRequest}
+                            feature={segmentChange.name}
```
Comment (kwasniew) [inline frontend/src/component/changeRequest/ChangeRequest/ChangeRequest.tsx:40]: Hmm. segmentChange.name is not a feature name, but a current segment name of the segment you're changing.

Comment (sjaanus) [reply frontend/src/component/changeRequest/ChangeRequest/ChangeRequest.tsx:40]: Yes, currently ChangeActions has actions for delete and edit. Edit needs a name of something. Currently providing segment name, but when we implement edit of segment change, then we need to refactor this a little.

**Review by kwasniew — APPROVED** [review]

**Pull request merged**